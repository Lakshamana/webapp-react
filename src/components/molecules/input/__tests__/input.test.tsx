import { shallow } from "setupTests";
import toJson from 'enzyme-to-json'
import { Input } from "../index";

const setup = () => {
const props = {};

 return {
   props,
   wrapper: shallow(<Input {...props} />)
 };
};

describe("Input test suit -", () => {
 it("Input renders without crashing given the required props", () => {
   const { wrapper } = setup();
   expect(wrapper.exists()).toBeTruthy();
   expect(toJson(wrapper)).toMatchSnapshot()
 });
});