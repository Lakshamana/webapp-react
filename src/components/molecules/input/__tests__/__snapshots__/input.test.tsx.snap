// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input test suit - Input renders without crashing given the required props 1`] = `
<InputWrapper
  background="#444444"
  border="1px solid #eaebeb"
  color="white"
  disabled={false}
  onChangeShowPassword={[Function]}
  placeholderColor="#F2FFF8"
  required={false}
  rightIcon=""
  type="text"
>
  <InputInline
    autoComplete="off"
    background="#444444"
    color="#fff"
    onKeyDown={[Function]}
    placeholderColor="#F2FFF8"
    type="text"
  />
</InputWrapper>
`;
